fe/src/main/java/org/apache/doris/analysis/CreateRoutineLoadStmt.java

```diff
@@ -328,6 +357,19 @@ private void checkJobProperties() throws UserException {
             timezone = ConnectContext.get().getSessionVariable().getTimeZone();
         }
         timezone = TimeUtils.checkTimeZoneValidAndStandardize(jobProperties.getOrDefault(LoadStmt.TIMEZONE, timezone));
+
+        format = jobProperties.get(FORMAT);
+        if (format != null) {
+            if (!format.equalsIgnoreCase("json")) {
+                format = "";// if it's not json, then it's mean csv and set empty
```

Comment:
I think we can just throw an exception here, means user specified a format which we don't support.